arch/riscv/lib/memcpy.S

```diff
@@ -89,6 +89,21 @@ ENTRY(memcpy)
 	/* Handle trailing misalignment */
 	beqz a2, 6f
 	add a3, a1, a2
+
+	/* Use word-oriented copy if co-aligned to word boundary */
+	or a5, a1, t6
+	or a5, a5, a3
+	andi a5, a5, 3
+	bnez a5, 5f
```

Comment:
Lastly, by the description, like it the patch was going to use lw/sw sequence in RV64, but didn't make it into this patch.